src/components/Navbar/NavbarForLargeScreen.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link, BrowserRouter as Router } from "react-router-dom";
+import { Button, Dropdown, DropdownButton } from "react-bootstrap";
+import { largeScreenNavbarData } from "./../../containers/Navbar/largeScreenNavbarData.js";
+
+const NavbarForLargeScreen = () => {
+  return (
+    <Router>
+      <div className="navbar">
+        <div className="navbarLogo">
+          <Link to="/">LOGO</Link>
+        </div>
+        <div className="navbarLinks">
+          {largeScreenNavbarData.map((info, index) => (
+            <Link to={info.target} className={info.cName} key={index}>
```

Comment:
lets no use index as keys https://dev.to/rajnishkatharotiya/you-shouldn-t-use-an-index-as-key-of-the-element-2jf

Reply:
@halitbatur Well I created specific ids for each user in data files then I changed index to info.id